README.md

```diff
@@ -1 +1,32 @@
-# review-past-project
\ No newline at end of file
+# review-past-project
+> This project is about Original Trombones musical instrument. The website shows   the features of the instrument and it different types with their prices.
+
+## Built With
+
+- HTML / CSS
+
+## How to run the program locally
+- Clone the project: ```https://github.com/mmsesay/review-past-project.git```
+- Navigate to the cloned project's directory
+- Locate the file ```index.hmtl``` and open it on your browser
+
+## Author
+
+👤 **Muhammad Mustapha Sesay**
+
+- GitHub: [@mmsesay](https://github.com/mmsesay)
+- Twitter: [@DeeMaejor](https://twitter.com/DeeMaejor)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/muhammad-m-sesay)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+- microverseinc for the readme template
```

Comment:
Please acknowledge the designers of the original template you used and it will be nice to add screenshots too.